### PR TITLE
Changelog 2.3 + CI failure fix 

### PR DESCRIPTION
**Pull request by daschuer**
This should hopefully fix CI issues in case the CHANGELOG.md has changed in a merge commit. 
Like this: 
https://github.com/mixxxdj/mixxx/runs/8264160246?check_suite_focus=true

### CODE REVIEW

**Comment by daschuer** [comment]
That was a hard nut. Hopefully with the latest changes to update_metainfo.py we will never see CI failures again cause by this.

**Comment by daschuer** [comment]
This is again up to date. I want to release 2.3.4 after this is merged. Please have a look.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,22 +7,29 @@
...
 * Numark DJ2Go2: Fix inverted tempo fader [#10852](https://github.com/mixxxdj/mixxx/pull/10852) [#10586](https://github.com/mixxxdj/mixxx/issues/10586)
+* Ableton Push: Show as one device [#10905](https://github.com/mixxxdj/mixxx/pull/10905)
 * Potmeters: Add support for arbitrary maximums in 7-/14-bit handlers from controller scripts [#4495](https://github.com/mixxxdj/mixxx/pull/4495)
 * Shade: Fix library sidebar splitter glitch [#4828](https://github.com/mixxxdj/mixxx/pull/4828) [lp:1979823](https://bugs.launchpad.net/mixxx/+bug/1979823)
 * MacOS builds: Perform ad-hoc signing of macOS bundle in Pull request and personal repositories [#4774](https://github.com/mixxxdj/mixxx/pull/4774)
 * Waveform: Avoid visual glitch with ranges < 1 px [#4804](https://github.com/mixxxdj/mixxx/pull/4804)
 * Traktor S3: Fix issues with sampler and hotcue buttons [#4676](https://github.com/mixxxdj/mixxx/pull/4676)
-* Build Mixxx on MacOS 11, replacing deprecated MacOS 10.15 [#4863](https://github.com/mixxxdj/mixxx/pull/4863)
+* Build Mixxx on macOS 11, replacing deprecated MacOS 10.15 [#4863](https://github.com/mixxxdj/mixxx/pull/4863)
```
Comment (fwcd) [inline CHANGELOG.md:16]: If we update the `macOS` capitalization there, we can do it for 10.15 too

```suggestion
* Build Mixxx on macOS 11, replacing deprecated macOS 10.15 [#4863](https://github.com/mixxxdj/mixxx/pull/4863)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,22 +7,29 @@
...
 * EQ preferences: properly restore 'One EQ for all decks' setting [#4886](https://github.com/mixxxdj/mixxx/pull/4886)
 * MC7000: Fix off-by-one indexing in the controller script [#4902](https://github.com/mixxxdj/mixxx/pull/4902)
 * Cover Art: Fix picking wrong cover file, when track file name contains extra dots [#4909](https://github.com/mixxxdj/mixxx/pull/4909)
-* Respect MusicBrainz rate limits [#10874](https://github.com/mixxxdj/mixxx/pull/10874) [#10795](https://github.com/mixxxdj/mixxx/issues/10795)
+* MusicBrainz: Respect rate limits [#10874](https://github.com/mixxxdj/mixxx/pull/10874) [#10795](https://github.com/mixxxdj/mixxx/issues/10795)
+* MusicBrainz: Stop fetching after closing the dialog [#10878](https://github.com/mixxxdj/mixxx/pull/10878) [#10877](https://github.com/mixxxdj/mixxx/issues/10877)
+* MacOs: Fix freezed skin control after Ctrl-Click [#10869](https://github.com/mixxxdj/mixxx/pull/10869) [10831](https://github.com/mixxxdj/mixxx/issues/10831)
```
Comment (fwcd) [inline CHANGELOG.md:22]: Same here:

```suggestion
* macOS: Fix freezed skin control after Ctrl-Click [#10869](https://github.com/mixxxdj/mixxx/pull/10869) [10831](https://github.com/mixxxdj/mixxx/issues/10831)
```

File: res/linux/org.mixxx.Mixxx.metainfo.xml [hunk 3]
```diff
@@ -146,7 +150,7 @@
    #4676
   </li>
   <li>
-   Build Mixxx on MacOS 11, replacing deprecated MacOS 10.15
+   Build Mixxx on macOS 11, replacing deprecated MacOS 10.15
```
Comment (fwcd) [inline res/linux/org.mixxx.Mixxx.metainfo.xml:153]:
```suggestion
   Build Mixxx on macOS 11, replacing deprecated macOS 10.15
```

**Comment by fwcd** [comment]
Out of curiosity: We build on macOS 11 now, but do we still target older macOS versions? According to the Mixxx homepage we support versions as far back as 10.12 with 2.3 and, at least until 2.4, we probably shouldn't drop any OS versions.

**Comment by fwcd** [comment]
Aand, sorry if this is not the right place to ask, but is there a tiny chance to get https://github.com/mixxxdj/mixxx/pull/4874 included in 2.3.4? This would really be an awesome fix.

**Comment by daschuer** [comment]
> Aand, sorry if this is not the right place to ask ...

I will have a look at the PR. In general we will move to a 6 Month release cycle, so that missing a deadline is not too bad. Else we will always have one last PR open.

**Comment by daschuer** [comment]
Done. @ronso0: merge?

**Comment by ywwg** [comment]
> Aand, sorry if this is not the right place to ask, but is there a tiny chance to get #4874 included in 2.3.4? This would really be an awesome fix

this is a controller fix so it's easier for us to include it in a new version.  I'll take a look and see if that's doable.

**Comment by ywwg** [comment]
> Aand, sorry if this is not the right place to ask, but is there a tiny chance to get #4874 included in 2.3.4? This would really be an awesome fix.

unfortunately, while that change does correct the behavior for that one controller, the bug affects all controllers.  We'd prefer to develop a solution for the deeper problem rather than patch over the symptom. Because that requires design work, it's not feasible to include the fix in 2.3.4

**Review by ywwg — APPROVED** [review]

**Pull request merged**